290. Word Pattern.py

```diff
@@ -0,0 +1,34 @@
+from collections import OrderedDict
```

Comment:
I don't think we need `OrderedDict` since we don't rely on order anywhere. Try using a normal dict.